Sources/ArgumentsUtils.swift

```diff
@@ -126,20 +134,24 @@ let bundledTemplatesPath = Path(ProcessInfo.processInfo.arguments[0]).parent() +
  - returns: The Path matching the template to find
  */
 func findTemplate(prefix: String, templateShortName: String, templateFullPath: String) throws -> Path {
-  guard templateFullPath.isEmpty else {
+  switch (templateFullPath, templateShortName) {
+  case ("", ""): throw TemplateError.noTemplateFound
```

Comment:
Embedded would mean (to me) that the templates are embedded in the binary, which they aren't. I think we can keep "bundled" for now.